README.md

```diff
@@ -262,7 +262,7 @@ Most dependencies can be installed using `apt-get install`:
 sudo apt-get install cmake g++ git automake libtool libgc-dev bison flex \
 libfl-dev libboost-dev libboost-iostreams-dev \
 libboost-graph-dev llvm pkg-config python3 python3-pip \
-tcpdump
+tcpdump libpcap-dev gcc-multilib
```

Comment:
These packages are required to build & test the whole compiler. Copy-pasting a single `apt-get` command is rather convenient and IMHO it would be nice if that was the only step required to get `make check` passing for the default build options (which include ebpf/ubpf) -- rather than digging through multiple backend-specific README files.

This is just my opinion based on my recent experience of bringing up this repo. Happy to change the patch as suggested if that is the consensus.

Reply:
Alternatively, we should check with CMake whether all necessary dependencies are available for the eBPF back end. If that is not the case throw an error. Or simply disable the eBPF back end by default. 

It is better to not pollute the top-level with back end specific dependencies and instructions. Especially, because we keep adding back ends. 

We should really think of adding a comprehensive install script for these things...

Reply:
I think you are also missing done libelf related packages

Reply:
Is there any kind of consensus on what we want in the top level README, vs. in back-end specific files here?

(a) One option: Top level README gives instructions that enable building _ALL_ back ends, if that is possible on a single system (I think it is, with the right prerequisites installed).

(b) Another option: Top level README gives instructions for building p4test _ONLY_.  Or at least a particular set of back ends that is explicitly listed as supported by the top level README instructions, with explicit pointers to back-end-specific READMEs where additional prerequisites are listed for each of them.

Any strong preferences?

Reply:
I personally prefer b) here. a) can turn messy really quickly. For the current back ends we have it already will be. The psa-eBPF backend, which uses NIKSS, and p4testgen already require a lot of dependencies to run, for example. You could make this even more complex with install instruction for all the different distributions we would need to write down etc.